Makefile

```diff
@@ -147,8 +147,8 @@ testcore: $(BUILD)/connect-core
 .PHONY: testnode
 testnode: $(BIN)/node16 $(BIN)/node18 $(BUILD)/connect-node-test
 	$(MAKE) crosstestserverrun
-	cd packages/connect-node-test && PATH="$(abspath $(BIN)):$(PATH)" NODE_TLS_REJECT_UNAUTHORIZED=0 node16 ../../node_modules/.bin/jasmine --config=jasmine.json
-	cd packages/connect-node-test && PATH="$(abspath $(BIN)):$(PATH)" NODE_TLS_REJECT_UNAUTHORIZED=0 node18 ../../node_modules/.bin/jasmine --config=jasmine.json
+	cd packages/connect-node-test && PATH="$(abspath $(BIN)):$(PATH)" node16 ../../node_modules/.bin/jasmine --config=jasmine.json
```

Comment:
We shouldn't ignore certificate errors globally in our tests. We now use a valid one for tests between @bufbuild/connect-node and @bufbuild/connect-node.

I wasn't able to configure the @bufbuild/connnect-node clients to work with the certificate used by connect-go. These client/server combinations now set the option `rejectUnauthorized: false`, so we only ignore errors for the specific combination. Added TODOs to investigate later.